### ISSUE TITLE
Is it possible to optionally display the copied and pasted text in the console window?

### ISSUE THREAD

**Issue by mdell-seradex** [Genymobile/scrcpy#3475]
Currently the console displays the fact that text was copied/synchronized between the PC and the Android device.
I was wondering if an option could be added that would display that text in the console?

Another option would be for it to be logged to a file.

I have looked at the Readme and FAQ, and at the command line parameters, but did not see anything like this.

Thank you.

**Comment by ssailly** [comment]
I just started working on this issue, and noticed there were two messages showing despite copying only once:
```
INFO: Device clipboard copied
DEBUG: Computer clipboard unchanged
```
However, computer clipboard is effectively changed, and this might correspond to two messages sent to the client. I will report it in a new issue and try to fix it.
